app/Photo.php

```diff
@@ -25,6 +26,12 @@ public static function getLastRecord()
         return self::orderBy('id', 'desc')->first();
     }
 
+    public static function getLastRowByTime($timeInMinutes)
+    {
+        $formatted_date = Carbon::now()->subMinutes($timeInMinutes)->toDateTimeString();
```

Comment:
inconsistent naming convention $formattedDate

Reply:
Check for other occurrences in code

Reply:
Yes, my mistake. I'm confused by my main work and this project. Edited